checks.d/yarn.py

```diff
@@ -196,9 +197,22 @@ def _yarn_app_metrics(self, rm_address, addl_tags):
 
                     for app_json in metrics_json['apps']['app']:
 
-                        app_name = app_json['name']
+                        tags = []
+
+                        for tag in app_tags:
+
+                            try:
+                                key, value = tag.split(':')
+                            except ValueError:
+                                self.log.error("Invalid value {0} for application_tag").format()
```

Comment:
Typo ? Should probably be something like:
`self.log.error("Invalid value {0} for application_tag".format(tag)`